rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/tests/activityList.additional.spec.ts

```diff
@@ -0,0 +1,271 @@
+import {
+  rundeckServiceMock,
+  rundeckClientMock,
+  axiosMock,
+  i18nMocks,
+  mockEventBus,
+} from "./mock";
+import { shallowMount, VueWrapper } from "@vue/test-utils";
+import ActivityList from "../activityList.vue";
+import ActivityFilter from "../activityFilter.vue";
+import OffsetPagination from "../../../../library/components/utils/OffsetPagination.vue";
+import { setupRundeckContext } from "./setupRundeckContext";
+import { GlobalOptions } from "./type";
+jest.mock("../../../../library/rundeckService", () => rundeckServiceMock);
+jest.mock("@rundeck/client", () => rundeckClientMock);
+jest.mock("axios", () => axiosMock);
+type ActivityListInstance = InstanceType<typeof ActivityList>;
+const shallowMountActivityList = async (
+  props = {},
+  globalOptions: GlobalOptions = {}
+) => {
+  const wrapper = shallowMount(ActivityList, {
+    props: {
+      eventBus: mockEventBus,
+      displayMode: "full",
+      ...props,
+    },
+    global: {
+      components: {
+        OffsetPagination,
+        ActivityFilter,
+      },
+      stubs: {
+        modal: true,
+        ActivityFilter: true,
+        Modal: {
+          template: "<div class='modal-stub'><slot></slot></div>",
+        },
+        ProgressBar: { template: "<div><slot></slot></div>" },
+        btn: {
+          template: `<button :disabled="disabled" data-test-id="activity-list-delete-selected-executions">Delete Selected Executions</button>`,
+          props: ["disabled"],
+        },
+
+        tooltip: true,
+        ...globalOptions.stubs,
```

Comment:
When using shallow mount, stubs are not needed

Reply:
> When using shallow mount, stubs are not needed

Thank you for your feedback. I understand that when using shallowMount, stubs are not necessary as child components are automatically stubbed. I will remove the stubs and run the tests again to ensure everything works as expected.